feature/policy_forwarding/otg_tests/mpls_gre_ipv4_encap_scale_test/mpls_gre_ipv4_encap_scale_test.go

```diff
@@ -0,0 +1,566 @@
+// Package mpls_gre_ipv4_encap_scale_test tests mplsogre encap functionality.
+package mpls_gre_ipv4_encap_scale_test
+
+import (
+	"fmt"
+	"testing"
+	"time"
+
+	"github.com/open-traffic-generator/snappi/gosnappi"
+	"github.com/openconfig/featureprofiles/internal/attrs"
+	"github.com/openconfig/featureprofiles/internal/cfgplugins"
+	"github.com/openconfig/featureprofiles/internal/deviations"
+	"github.com/openconfig/featureprofiles/internal/fptest"
+	"github.com/openconfig/featureprofiles/internal/iputil"
+	otgconfighelpers "github.com/openconfig/featureprofiles/internal/otg_helpers/otg_config_helpers"
+	otgvalidationhelpers "github.com/openconfig/featureprofiles/internal/otg_helpers/otg_validation_helpers"
+	packetvalidationhelpers "github.com/openconfig/featureprofiles/internal/otg_helpers/packetvalidationhelpers"
```

Comment:
Sync client and delete unneeded alias - these aliases are already the existing package names.

https://github.com/openconfig/featureprofiles/blob/634c13fb50263b4fd7b01ef82c462dfebcf680b1/internal/otg_helpers/otg_config_helpers/otgconfighelpers.go#L1

https://github.com/openconfig/featureprofiles/blob/634c13fb50263b4fd7b01ef82c462dfebcf680b1/internal/otg_helpers/otg_validation_helpers/otgvalidationhelpers.go#L2

https://github.com/openconfig/featureprofiles/blob/634c13fb50263b4fd7b01ef82c462dfebcf680b1/internal/otg_helpers/packetvalidationhelpers/packetvalidationhelpers.go#L2

```suggestion
	"github.com/openconfig/featureprofiles/internal/otg_helpers/otg_config_helpers"
	"github.com/openconfig/featureprofiles/internal/otg_helpers/otg_validation_helpers"
	"github.com/openconfig/featureprofiles/internal/otg_helpers/packetvalidationhelpers"
```

Reply:
I'm unable to rename the files `otg_config_helpers` and `otg_validation_helpers` at the moment, as they are already being imported by other users. I'll address this in a separate PR and update the relevant tests accordingly.